sdk/test-utils/perfstress/src/program.ts

```diff
@@ -217,7 +217,7 @@ export class PerfStressProgram {
     // of operations running.
     const millisecondsToLog = Number(this.parsedDefaultOptions["milliseconds-to-log"].value);
     console.log(
-      `\n=== ${title} mode, iteration ${iterationIndex}. Logs every ${millisecondsToLog /
+      `\n=== ${title} mode, iteration ${iterationIndex + 1}. Logs every ${millisecondsToLog /
```

Comment:
At line 273, we do `+1` but not here.. So, looking good! 👍

Hope your automation doesn't break with this change, @mikeharder.

Reply:
It should be fine.  The automation searches the console output using a regex for the final results, so most changes or additions to the output shouldn't break it.  Longer-term we're going to add a JSON output format to each framework so the automation doesn't need to screen-scrape.